add AI Policy (#5006)
PrismLauncher/PrismLauncher: CONTRIBUTING.md

# Contributions Guidelines

## Restrictions on Generative AI Usage (AI Policy)

> 

Reviewer: ```suggestion
Adapted from [matplotlib's docs](https://matplotlib.org/devdocs/devel/contribute.html#generative-ai):
```

> Adapted from [matplotlib's docs](https://matplotlib.org/devdocs/devel/contribute.html#generative-ai):

Reviewer 2: ```suggestion
Adapted from [matplotlib's docs](https://matplotlib.org/devdocs/devel/contribute.html#generative-ai):

```
add an empty line to separate the paragraph

We expect authentic engagement in our community.

- Do not post output from Large Language Models or similar generative AI as comments on GitHub or our discord server, as such comments tend to be formulaic and low-quality content.
- If you use generative AI tools as an aid in developing code or documentation changes, ensure that you fully understand the proposed changes and can explain why they are the correct approach.

Make sure you have added value based on your personal competency to your contributions. 
Just taking some input, feeding it to an AI and posting the result is not of value to the project. 
To preserve precious core developer capacity, we reserve the right to rigorously reject seemingly AI generated low-value contributions.

> 

Reviewer 2: ````suggestion
Adapted from the [Linux Kernal](https://www.kernel.org/doc./html/next/process/coding-assistants.html):

### Signed-off-by and Developer Certificate of Origin 

AI agents MUST NOT add Signed-off-by tags. Only humans can legally certify the Developer Certificate of Origin (DCO). The human submitter is responsible for:

  - Reviewing all AI-generated code
  - Ensuring compliance with licensing requirements
  - Adding their own Signed-off-by tag to certify the DCO
  - Taking full responsibility for the contribution
  
See [Signing your work](#signing-your-work) for more information.

### Attribution

When AI tools contribute to development, proper attribution helps track the evolving role of AI in the development process. Contributions should include an Assisted-by tag in the commit message with the following format:

```
Assisted-by: AGENT_NAME:MODEL_VERSION [TOOL1] [TOOL2]
```

Where:

  - `AGENT_NAME` is the name of the AI tool or framework
  - `MODEL_VERSION` is the specific model version used
  - `[TOOL1] [TOOL2]` are optional specialized analysis tools used (e.g., coccinelle, sparse, smatch, clang-tidy)

Basic development tools (git, gcc, make, editors) should not be listed.

Example:

```
Assisted-by: Claude:claude-3-opus coccinelle sparse
```

````

Pull in info about DCO and adding AI attribution form Linux Kernal project

# Contributions Guidelines

## Restrictions on Generative AI Usage (AI Policy)
Adapted from [matplotlib's docs](https://matplotlib.org/devdocs/devel/contribute.html#generative-ai):

We expect authentic engagement in our community.

- Do not post output from Large Language Models or similar generative AI as comments on GitHub or our discord server, as such comments tend to be formulaic and low-quality content.
- If you use generative AI tools as an aid in developing code or documentation changes, ensure that you fully understand the proposed changes and can explain why they are the correct approach.

Make sure you have added value based on your personal competency to your contributions. 
Just taking some input, feeding it to an AI and posting the result is not of value to the project. 
To preserve precious core developer capacity, we reserve the right to rigorously reject seemingly AI generated low-value contributions.

> Adapted from the [Linux Kernal](https://www.kernel.org/doc./html/next/process/coding-assistants.html):

Reviewer 2: ```suggestion

Adapted from the [Linux Kernal](https://www.kernel.org/doc./html/next/process/coding-assistants.html):
```
extra new line needed, my bad

# Contributions Guidelines

## Restrictions on Generative AI Usage (AI Policy)

> Adapted from [matplotlib's docs](https://matplotlib.org/devdocs/devel/contribute.html#generative-ai):

Reviewer 3: ```suggestion
> [!NOTE]
> The following is adapted from [matplotlib's contributing guide](https://matplotlib.org/devdocs/devel/contribute.html#generative-ai) and the [Linux Kernel policy guide](https://www.kernel.org/doc./html/next/process/coding-assistants.html)
```

Slightly better formatting for both of these

# Contributions Guidelines

## Restrictions on Generative AI Usage (AI Policy)
> [!NOTE]
> The following is adapted from [matplotlib's contributing guide](https://matplotlib.org/devdocs/devel/contribute.html#generative-ai) and the [Linux Kernel policy guide](https://www.kernel.org/doc./html/next/process/coding-assistants.html)

We expect authentic engagement in our community.

- Do not post output from Large Language Models or similar generative AI as comments on GitHub or our discord server, as such comments tend to be formulaic and low-quality content.
- If you use generative AI tools as an aid in developing code or documentation changes, ensure that you fully understand the proposed changes and can explain why they are the correct approach.

Make sure you have added value based on your personal competency to your contributions. 
Just taking some input, feeding it to an AI and posting the result is not of value to the project. 
To preserve precious core developer capacity, we reserve the right to rigorously reject seemingly AI generated low-value contributions.

> [!NOTE]

> > The following is adapted from the [Linux Kernel](https://www.kernel.org/doc./html/next/process/coding-assistants.html):

Reviewer 3: This is already covered by the above, it doesn't need to be here

Author: oh you did add the link there, right. whole line can go.
